

root@csst-06:/home/ubuntu/laspdev# 
root@csst-06:/home/ubuntu/laspdev# python /home/ubuntu/laspdev/utility/setup_lasp.py d4
d4
ip:10.0.0.14
node:d
)2> LASPVIN no peer to subscribe case 2 
lasp_peer_service:join('a@10.0.0.11').
ok
(d@10.0.0.14)3> LASPVIN no peer to subscribe case 2 
15:29:54.329 [error] LASPVIN received find_sub Id: "c@10.0.0.13c1" From: 'a@10.0.0.11' 
15:29:54.331 [error] LASPVIN Peer 'c@10.0.0.13' is Source of Req.. Skipping 
15:29:54.331 [error] LASPVIN Request forwarded~n
15:29:55.331 [error] LASPVIN False call
15:30:01.359 [error] LASPVIN received find_sub Id: "c@10.0.0.13c1" From: 'c@10.0.0.13' 
15:30:01.359 [error] LASPVIN Request forwarded~n
15:30:32.442 [warning] global: 'd@10.0.0.14' failed to connect to 'a@12.0.0.11'
15:30:32.443 [warning] global: 'd@10.0.0.14' failed to connect to 'a@11.0.0.11'
15:31:04.084 [error] LASPVIN Received delta From='a@10.0.0.11' at TimeStamp="2020-06-06T15:31:04.842" Took=41714 microseconds 
